fftoml/fftoml.go

```diff
@@ -6,27 +6,69 @@ import (
 	"io"
 	"strconv"
 
-	"github.com/BurntSushi/toml"
+	"github.com/pelletier/go-toml"
 	"github.com/peterbourgon/ff"
 )
 
 // Parser is a parser for TOML file format. Flags and their values are read
 // from the key/value pairs defined in the config file.
 func Parser(r io.Reader, set func(name, value string) error) error {
-	var m map[string]interface{}
-	_, err := toml.DecodeReader(r, &m)
-	if err != nil {
-		return ParseError{Inner: err}
+	return ParserWith()(r, set)
+}
+
+type config struct {
+	separator string
+}
+
+// Option is a function which configures a fftoml.ConfigFileParser
+type Option func(*config)
+
+// FlagSeparator is an option which configures a separator
+// to use when constructing a flag name
```

Comment:
Please make this comment a bit more descriptive, and add a sentence like "The default separator is a period `.`."